nav2_dwb_controller/dwb_plugins/include/dwb_plugins/kinematic_parameters.hpp

```diff
@@ -113,13 +107,34 @@ class KinematicParameters
   double min_speed_xy_sq_{0};
   double max_speed_xy_sq_{0};
 
-  void reconfigureCB();
+  friend class KinematicsHandler;
```

Comment:
Can you put this at the top of the struct? It'll make that more clear to passive readers

Reply:
For sure, I'm on it.